tests/baselines/reference/ArrowFunction1.types

```diff
@@ -1,7 +1,7 @@
 === tests/cases/conformance/parser/ecmascript5/ErrorRecovery/ArrowFunctions/ArrowFunction1.ts ===
 var v = (a: ) => {
->v : (a: any) => void
->(a: ) => {   } : (a: any) => void
+>v : (a: ) => void
+>(a: ) => {   } : (a: ) => void

```

Comment:
Uh.

Reply:
`MissingNode` is a `TypeNode` we produce :P

We can remap it like we do jsdoc.